### ISSUE TITLE
.accessibilityrc wont work after grunt-accessibility update 

### ISSUE THREAD

**Issue by stephanfriedrich** [yargalot/AccessSniff#18]
Hi, i updated my grunt-accessibility Module.
**unfortunately my ignore values wont work, i dont know whats wrong**.
It seems it wont read the content of .accessibilityrc

**my gruntfile Task-Options:**

```
accessibility: {
     options: {
       accessibilityLevel: 'WCAG2AA',
       accessibilityrc: true,
       domElement: true,
       reportLevels: {
         notice: false,
         warning: true,
         error: true
       }
  }
```

**example of my .accessibilityrc:**

```
{
  "ignore": [
    "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
    "WCAG2AA.Principle1.Guideline1_3.1_3_1.H48",
    "WCAG2AA.Principle1.Guideline1_3.1_3_1.H39.3.NoCaption",
  ]
}

```

**old:**
grunt-accessibility@3.5.1
├─ access-sniff@1.1.2

Verbose Output:
`Options: ignore=[], verbose, force=false, domElement, reportType=null, reportLocation="reports", accessibilityrc, accessibilityLevel="WCAG2AA", reportLevels={"notice":false,"warning":true,"error":true}`

**new**:
grunt-accessibility@4.1.0
 └─ access-sniff@2.4.4

`Options: accessibilityLevel="WCAG2AA", accessibilityrc, domElement, reportLevels={"notice":false,"warning":true,"error":true}`

**Comment by stephanfriedrich** [comment]
BTW.:
if i put the ignore values direct into my gruntfile.js, it works.

```
accessibility: {
     options: {
       accessibilityLevel: 'WCAG2AA',
       "ignore": [
         "WCAG2AA.Principle4.Guideline4_1.4_1_1.F77",
         "WCAG2AA.Principle1.Guideline1_3.1_3_1.H48",
         "WCAG2AA.Principle1.Guideline1_3.1_3_1.H39.3.NoCaption",
       ],
       domElement: true,
       reportLevels: {
         notice: false,
         warning: true,
         error: true
       }
  }
```

**Comment by jgrassis** [comment]
Same here, but with Gulp. Anything I put directly into the gulpfile works fine, but any options I try to set via the .accessibilityrc file is ignored.

**Comment by yargalot** [comment]
fixed in 3.0
